openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -94,15 +96,25 @@ def _create_from_course(course):
         except ValueError:
             lowest_passing_grade = None
 
+        display_name = course.display_name
+        start = course.start
+        end = course.end
+        if isinstance(course.id, CCXLocator):
+            from ccx.utils import get_ccx_from_ccx_locator  # pylint: disable=import-error
+            ccx = get_ccx_from_ccx_locator(course.id)
+            display_name = ccx.display_name
+            start = ccx.start
+            end = ccx.due
```

Comment:
While I think this would be possible, I think it could be a bit tricky. At this point I'm not even sure how where I would start with that. Could we perhaps leave this out of scope of this PR for now?